migration/migrate.go

```diff
@@ -0,0 +1,45 @@
+package migration
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/go-devs-ua/octagon/app/repository/pg"
+)
+
+// Execute migrations.
+// Use "migrate-up" argument after run of program. It will create new table
+// Or use "migrate-down" - that will remove table
+// Example: go run main.go migrate-up
+func Migrate(repo *pg.Repo) error {
+	for _, i := range os.Args {
```

Comment:
It is more appropriate to use `arg` or `a` rather than `i`. It is confusing because I get used to naming index-related variables i, j, k ....

Reply:
This is a common variable name when using range. The `i` is meaning "item" of some slice/array. I do not see the problem with `i`

Reply:
`for _, i := range os.Args` it very similar to `for i, _ := range os.Args`

You more often see i as index than as item. 
Item isn't long word that can't be used here)

We also have another good practice - give items over that you are iterating meaningful names.
In our case, arg looks not better than v but better than i.

If you iterate over users slice - you would prefer calling each item as u or user instead of i or v.

Reply:
=) There is no point to discus. I accepted and fixed it